lib/server.coffee

```diff
@@ -126,7 +124,7 @@ WebApp.connectHandlers.use (req, res, next) ->
 
 		Spiderable.originalRequest = req
 		url 		= Spiderable._urlForPhantom Meteor.absoluteUrl(), req.url
-		hash 		= SHA256 url
+		hash 		= hashCode url
```

Comment:
I don't think it makes sense to remove SHA256 and implement our own hashCode. If the fear is performance (which is very unlikely), then use a less intensive, standard hasher.
I recommend reverting this line.